docs/setup/README.md

```diff
@@ -0,0 +1,21 @@
+# Installation
+
+### Requirements
+
+* Node.js 4 or higher

```

Comment:
I believe we are actually using Node v6 or higher (check the package.json `"engines"`).